app/bundles/PluginBundle/Form/Type/FieldsTypeTrait.php

```diff
@@ -172,7 +172,8 @@ function (FormEvent $event) use ($options, $integrationFields, $mauticFields, $f
                     }
                     if (!$fieldObject) {
                         $contactLink['mauticContactTimelineLink'] = $this->translator->trans('mautic.plugin.integration.contact.timeline.link');
-                        $mauticFields = array_merge($mauticFields, $contactLink);
+                        $isContactable['mauticContactIsContactable'] = $this->translator->trans('mautic.plugin.integration.contact.donotcontact');
```

Comment:
Search and replace the uses of `mauticContactIsContactable` to be `mauticContactIsContactableByEmail` Again the idea is to allow us to add support for other channels later.